aiobotocore/endpoint.py

```diff
@@ -37,8 +48,9 @@ async def convert_to_response_dict(http_response, operation_model):
         * body (string or file-like object)
 
     """
-    response_dict = {
-        'headers': http_response.headers,
+    headers = http_response.headers
+    response_dict: dict[str, Any] = {
+        'headers': headers,
```

Comment:
ya these type of code changes should be to a minimum, we compare this to: https://github.com/boto/botocore/blob/develop/botocore/endpoint.py#L42